conference.js

```diff
@@ -3059,13 +3060,14 @@ export default {
      * Leaves the room.
      *
      * @param {boolean} doDisconnect - Whether leaving the room should also terminate the connection.
+     * @param {string} reason - reason for leaving the room.
      * @returns {Promise}
      */
-    async leaveRoom(doDisconnect = true) {
+    async leaveRoom(doDisconnect = true, reason = '') {
         APP.store.dispatch(conferenceWillLeave(room));
 
         if (room && room.isJoined()) {
-            return room.leave().finally(() => {
+            return room.leave(doDisconnect, reason).finally(() => {
```

Comment:
room.leave only takes the reason, not `doDisconnect`.

Reply:
room in this case than it's conference, right? (https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConference.js#L633) 
I will remove "doDisconnect"